utilities/simulator_cache/cache_simulator.cc

```diff
@@ -110,19 +110,22 @@ void PrioritizedCacheSimulator::Access(const BlockCacheTraceRecord& access) {
 std::string HybridRowBlockCacheSimulator::ComputeRowKey(
     const BlockCacheTraceRecord& access) {
   assert(access.get_id != BlockCacheTraceHelper::kReservedGetId);
-  Slice key;
-  if (access.referenced_key_exist_in_block == Boolean::kTrue) {
-    key = ExtractUserKey(access.referenced_key);
-  } else {
-    key = access.referenced_key;
-  }
-  return std::to_string(access.sst_fd_number) + "_" + key.ToString();
+  Slice key = ExtractUserKey(access.referenced_key);
+  uint64_t seq_no = access.is_snapshot_get == Boolean::kFalse
+                        ? 0
```

Comment:
1. The code always logs the sequence number right now. 
2. The problem of using the latest sequence number is that cached key-value pair is invalidated every time the sequence number increases. 

For example, the latest version of (K1,V1)'s has seq number as 1 in a SST file and is in the cache. Future Get with seq number 2,3,4,5 should continue to fetch (K1,V1) from the cache. If we tag the latest seq number of Get in the key, these gets will observe misses instead of hits.